std-bits/tableau/src/main/java/org/enso/tableau/HyperReader.java

```diff
@@ -0,0 +1,209 @@
+package org.enso.tableau;
+
+import com.tableau.hyperapi.Catalog;
+import com.tableau.hyperapi.Connection;
+import com.tableau.hyperapi.HyperProcess;
+import com.tableau.hyperapi.SchemaName;
+import com.tableau.hyperapi.TableName;
+import com.tableau.hyperapi.Telemetry;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.channels.Channels;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.IntStream;
+import org.enso.table.data.table.Column;
+import org.enso.table.problems.ProblemAggregator;
+
+// ** Class responsible for reading from Tableau Hyper files. */
```

Comment:
```suggestion
/** Class responsible for reading from Tableau Hyper files. */
```

I imagine this was supposed to be a block comment?